packages/babel-helpers/src/helpers/replaceTsImportExt.ts

```diff
@@ -0,0 +1,7 @@
+/* @minVersion 7.25.7 */
+
+export default function replaceTsImportExt(source: string) {
+  return /[\\/]/.test(source)
+    ? source.replace(/(\.[mc]?)ts$/, "$1js").replace(/\.tsx$/, ".js")
+    : source;
```

Comment:
This could probably be shorter:
```js
return String(source).replace(/([\\/].*\.[mc]?)tsx?$, "$1js");
```
and it becomes short enough that we could even just inline it in the code without a helper :P